src/librekey/rnp_key_store.cpp

```diff
@@ -717,8 +698,8 @@ rnp_key_store_get_key_by_id(pgp_io_t *             io,
          key_item = list_next(key_item)) {
         pgp_key_t *key = (pgp_key_t *) key_item;
         if (rnp_get_debug(__FILE__)) {
-            hexdump(io->errs, "keyring keyid", key->keyid, PGP_KEY_ID_SIZE);
-            hexdump(io->errs, "keyid", keyid, PGP_KEY_ID_SIZE);
+            hexdump(stderr, "keyring keyid", key->keyid, PGP_KEY_ID_SIZE);
+            hexdump(stderr, "keyid", keyid, PGP_KEY_ID_SIZE);
```

Comment:
I feel like we want a `RNP_LOG_BIN` for things like this.

Reply:
I added RNP_DHEX(), which also wraps if (rnp_get_debug(...))